.github/actions/run-bwc-suite/action.yaml

```diff
@@ -14,6 +14,18 @@ inputs:
     description: 'The name of the artifacts for this run, e.g. "BWC-2.1-to-2.4-results"'
     required: true
 
+  use-https:
+    description: 'Use HTTPS for wait for cluster health check in testClusters'
+    required: true
```

Comment:
Seems like we only run with https, lets not make this a parameter - unless you think we should run the BWC test with additional configurations?

Reply:
done